scripts/ai/AIDriveStrategyBunkerSilo.lua

```diff
@@ -579,8 +579,10 @@ function AIDriveStrategyBunkerSilo:startCourseWithPathfinding(course, ix, isReve
             setRotation(self.pathfinderNode.node, 0, yRot + math.pi, 0)
         end
 
-        self.pathfinder, done, path = PathfinderUtil.startPathfindingFromVehicleToNode(self.vehicle, self.pathfinderNode.node,
-            0, 0, true)
+        local context = PathfinderContext(self.vehicle):allowReverse(true):offFieldPenalty(0) context:maxFruitPercent(math.huge)
```

Comment:
``context:maxFruitPercent(math.huge)`` Copy/paste error or intentionally?

Reply:
Good catch. What should it be? In startPathfindingFromVehicleToNode() it used to be `maxFruitPercent or (settings.avoidFruit:getValue() and 50 or math.huge)` ?

Reply:
(So here, the equivalent would be `settings.avoidFruit:getValue() and 50 or math.huge`)

Reply:
Maybe we could add another simplified true or false behaviour for the fruit avoidence.
Then we could pass the setting value into it or use a strategy function to return true or false to the `context:avoidFruit()`.
`lua
PathfinderContext.DEFAULT_FRUIT_AVOIDENCE_PERCENTAGE = 50
--- Disables the fruit avoidance
function PathfinderContext:avoidFruit(avoidFruit[true/false])
    if avoidFruit then 
          self._maxFruitPercent = self.DEFAULT_FRUIT_AVOIDENCE_PERCENTAGE 
    else
          self._maxFruitPercent = math.huge
    end
    return self
end
`

Reply:
We already have an ignoreFruit()